uqcsbot/scripts/welcome.py

```diff
@@ -0,0 +1,39 @@
+# Welcomes new users to UQCS Slack and check for member milestones
+from uqcsbot import bot
+import asyncio
+
+MEMBER_MILESTONE = 50;  # Number of members between posting a celebration
+MESSAGE_PAUSE = 2500 / 1000;   # Number of seconds between sending bot messages
```

Comment:
Can we just put 2.5 here? It took be a bit to realise that you were doing milliseconds then converting to seconds.

Reply:
Yeah fair, I just copy-pasted from the old code where JS uses milliseconds